services/ui_backend_service/api/admin.py

```diff
@@ -0,0 +1,185 @@
+import os
+import hashlib
+import json
+import time
+from multidict import MultiDict
+from services.utils import (
+    web_response, METADATA_SERVICE_VERSION,
+    METADATA_SERVICE_HEADER, SERVICE_COMMIT_HASH,
+    SERVICE_BUILD_TIMESTAMP
+)
+from aiohttp import web
+
+UI_SERVICE_VERSION = "{metadata_v}-{timestamp}-{commit}".format(
+    metadata_v=METADATA_SERVICE_VERSION,
+    timestamp=SERVICE_BUILD_TIMESTAMP or "",
+    commit=SERVICE_COMMIT_HASH or ""
+)
```

Comment:
we need to come up with something else for the version exposure, if shared utils will get changes to exposing metadata service version.